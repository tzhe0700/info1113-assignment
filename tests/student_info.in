ADD STUDENT Harry Potter
STUDENT 100000
STUDENT 99999
EXIT

