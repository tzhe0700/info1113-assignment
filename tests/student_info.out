Success.
100000: Harry Potter
No such student in system.

